run_agents.py

```diff
@@ -1,21 +0,0 @@
-import numpy as np
```

Comment:
Nit: consider naming this file just `run.py`. That's a name most people would look for, and it avoids confusion that `rl_agents.py` and `run_agents.py` might be two different kinds of agents.